azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/toolkit/lib/auth/Account.java

```diff
@@ -132,7 +126,7 @@ public void selectSubscription(List<String> selectedSubscriptionIds) {
     protected abstract Mono<TokenCredentialManager> createTokenCredentialManager();
 
     public Mono<Boolean> checkAvailable() {
-        return loginStep1();
+        return preLoginCheck().doOnSuccess(avail -> this.entity.setAvailable(avail));
```

Comment:
is there any difference between `checkAvailable` and the so-called `preLoginCheck`?